Makefile

```diff
@@ -7,7 +7,10 @@ SLOW_TEST_THRESHOLD=1 # seconds
 VERSION=$(shell cat connectors/VERSION)
 
 
-bin/python:
+config:
```

Comment:
You might wanna change it to `config.yml` - this way the command will not run if `config.yml` is already in-place

Reply:
Oh interesting, is that a Makefile feature? I'll admit that I don't know a lot about `make`. 

It already doesn't overwrite the file if it exists. The command is:
```
- cp -n config.yml.example config.yml
```

the `-n` is "no clobber" which means that if `config.yml` exists, terminate. Unfortunately it terminates with an error, so the `-` before `cp` tells make to ignore errors in this step and proceed.

I've tested pretty extensively to ensure that the config file doesn't get overwritten.

Reply:
Yup, dependencies in Makefiles are actually dependencies on files, like here: https://www.tutorialspoint.com/makefile/makefile_dependencies.htm

It's not a biggie not to do, we can fix in subsequent prs :)